Improve ephemerality docs, adding `remove-stale-node` (#20057)
netdata/netdata: src/cli/README.md

| `remove-stale-node <node_id \| machine_guid \| hostname \| ALL_NODES>` | Marks one or all disconnected nodes, including virtual nodes, as [ephemeral](/docs/nodes-ephemerality.md), and removes them so that they are no longer available for queries, from both this Netdata Agent dashboard and Netdata Cloud. This is useful to remove decommissioned nodes that have stopped streaming, but for which the Agent still holds metric data.[^1][^3]|
| `version`                                                              | Display the Netdata Agent version.                                                                                                                                               |

See also the Netdata daemon [command line options](/src/daemon/README.md#command-line-options).

[^1]: This also clears any active alerts.
[^2]: The Agent may be configured to [automatically clean up ephemeral nodes](/docs/nodes-ephemerality.md#automatic-ephemeral-nodes-cleanup) when they have been offline for longer than the configured time interval.

> [^3]: If a node is represented by multiple Parent Agents in a HA setup, this command must be issued on all these Parent Agents.

Reviewer: ```suggestion
[^3]: If a node is represented by multiple Parent Agents in a HA setup, this command must be executed on each Parent Agent.
```
